# Describes pointnormals of a pointcloud
Header header
geometry_msgs/Point32[] points
geometry_msgs/Point32[] normals
